content/en/gitops.md

```diff
@@ -0,0 +1,40 @@
+---
+title: GitOps
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+
+GitOps is a set of best practices based on [shared principles](https://opengitops.dev/) applied to managing and operating a cloud-native infrastructure and applications. 
+The GitOps approach is used to strategize a cohesive workflow that leverages Git as the "single source of truth" and extends this practice to an application's configuration, infrastructure, and operational procedures. 
+The GitOps controller automates matching the Git state with the system state in a bidirectional manner.
+
+## Problem it addresses
+
+Some existing processes for infrastructure configuration management can face challenges, such as:
+
+- Failed deployments
+
+Relying on a disaster recovery strategy with manual or semi-manual processes isn't always reliable. 
+
+- Configuration drift
+
+This approach can create consequences, such as security vulnerabilities, high costs due to lack of efficiencies resulting in stale workloads, and risking downtime in an application or service because it's not scaling enough or is scaling too frequently.
+
+- Poor Infrastructure design
+
+Legacy infrastructure designs limit developers to access or understand large-scale systems that limit their productivity.
+
+- Missing documentation or unknown development history
+
+Many teams depend on working practices developed over decades and document changes in a centrally maintained wiki that is uncontrolled and not maintained.
+
+Adopting a GitOps workflow can help alleviate some of these issues.
+
+## How it helps
```

Comment:
Another reason is to enhance the security posture of an organisation. Using GitOps, organisations subscribe to a `pull` based approach as opposed to the `push` based approach that has more commonly been followed. Therefore, if a CI system (Jenkins etc) is compromised, the attack surface is now smaller because CI and CD are separated - as traditionally people deploy to Kubernetes via their CI systems using push strategies.

Reply:
@ChrisJBurns - I hesitate to focus on a pull/push-based operation approach only and mention Jenkins because this limits the context to remain vast for the audience who utilize other tools and approaches for their GitOps workflow.

Reply:
@hseligson1 really great start here but can we try and bring the entire problem section down to 3-5 lines.

Reply:
That's a fair point!